src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = (props) => {
```

Comment:
destruct person here, and below you can write 

`const {
    name,
    age,
    isMarried,
    partnerName,
    sex,
  } = person;`